js/repl/Worker.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import compile from "./compile";
+import registerPromiseWorker from "promise-worker/register";
+
+declare var Babel: any;
```

Comment:
Do webpack `externals` not work in a web worker? `require('babel-standalone')` should be the same as referencing the global `Babel`.

Reply:
Hm. Babel (or `babel-standalone`) isn't one of the things we alias though, is it?

Reply:
Oh good catch, we don't alias it because it's lazy loaded. Sorry for the confusion @bvaughn  :D